train.py

```diff
@@ -50,22 +49,21 @@
 arg_parser.add_argument("--spatial_backbone", type=str, default="mean_pixel")
 arg_parser.add_argument("--temporal_backbone", type=str, default="LSTM")
 arg_parser.add_argument("--image_size", type=int, default=32)
-arg_parser.add_argument("--save_model_validation_threshold", type=float, default=0.8)
-arg_parser.add_argument("--pse_sample_size", type=int, default=64)
-arg_parser.add_argument("--validation_split", type=float, default=0.25)
-arg_parser.add_argument("--skip_bands", dest="include_bands", action="store_false")
-arg_parser.set_defaults(include_bands=True)
-arg_parser.add_argument("--skip_cloud", dest="include_cloud", action="store_false")
-arg_parser.set_defaults(include_cloud=True)
-arg_parser.add_argument("--skip_ndvi", dest="include_ndvi", action="store_false")
-arg_parser.set_defaults(include_ndvi=True)
+arg_parser.add_argument("--save_model_threshold", type=float, default=0.8)
+arg_parser.add_argument("--pse_sample_size", type=int, default=32)
+arg_parser.add_argument("--validation_split", type=float, default=0.2)
+arg_parser.add_argument("--split_by", type=str, default="longitude", help="latitude or longitude")
+arg_parser.add_argument("--include_bands", type=bool, default=True)
+arg_parser.add_argument("--include_cloud", type=bool, default=True)
+arg_parser.add_argument("--include_ndvi", type=bool, default=False)
```

Comment:
Can I ask why we are not including NDVI by default anymore?

Reply:
I had a couple intermediate results which did worse with NDVI, but after looking at the most recent top runs for planet daily they do have NDVI so I included it back in (new commit).